\documentclass[draft]{scrbook}

% PACKAGES
\usepackage{typearea}
\usepackage{hyperref}
\usepackage[svgnames]{xcolor}
\usepackage{fontspec}
\usepackage{graphicx}
\usepackage{csquotes}
\usepackage[backend=biber, style=nature,maxcitenames=99, maxbibnames=99]{biblatex}
\usepackage[acronyms]{glossaries}
\usepackage{enumitem}
\usepackage{xparse}
\usepackage{cleveref}
\usepackage{siunitx}

% CUSTOM COMMANDS
\DeclareDocumentCommand{\newdualentry}{ O{} O{} m m m m } {
  \newglossaryentry{gls-#3}{name={#5},text={#5\glsadd{#3}},
    description={#6},#1
  }
  \makeglossaries
  \newacronym[see={[Glossary:]{gls-#3}},#2]{#3}{#4}{#5\glsadd{gls-#3}}
}

% SETTINGS
\definecolor{UppsalaRed}{RGB}{182,57,77}
\hypersetup{
    colorlinks=true,
    linkcolor=UppsalaRed,
    urlcolor=UppsalaRed,
    filecolor=UppsalaRed,
    citecolor=UppsalaRed
}
\defaultfontfeatures{Ligatures={Required, Common, TeX}}
\addbibresource{Half-Time Thesis.bib}
\DeclareSIUnit{\litre}{\ell}
\sisetup{
    per-mode = symbol
}

% METADATA
\title{The Effect of Common and Rare Genetic Variants on the Pathogenesis of Common Complex Diseases}
\author{Daniel Schmitz}
\date{2021-10-15}

% GLOSSARY
\makeglossaries
\newacronym{snp}{SNP}{single-nucleotide polymorphism}
\newacronym{cnv}{CNV}{copy-number variation}
\newacronym[
    plural=GWAS,
    firstplural=genome-wide association studies (GWAS)
    ]{gwas}{GWAS}{genome-wide association study}
\newacronym{ld}{LD}{linkage disequilibrium}
\newacronym{nsphs}{NSPHS}{Northern Swedish Population Health Study}
\newacronym{pea}{PEA}{protein extension assay}
\newacronym{ukb}{UKB}{UK Biobank}
\newacronym{wgs}{WGS}{whole-genome sequencing}
\newacronym{smrt}{SMRT}{Single-Molecule Real-Time Sequencing}
\newacronym{t2d}{T2D}{type 2 diabetes}
\newacronym{bmd}{BMD}{bone mineral density}
\newacronym{iv}{IV}{instrumental variable}
\newacronym{maf}{MAF}{minor allele frequency}
\newacronym{qc}{QC}{quality control}
\newacronym{bmi}{BMI}{body-mass index}
\newacronym{hrt}{HRT}{hormone-replacement therapy}
\newacronym{oc}{OC}{oral contraceptive}
\newacronym{pc}{PC}{principal component}
\newacronym{shbg}{SHBG}{sex hormone-binding globulin}
\newacronym[
    plural = eQTL,
    firstplural = expression quantitative trait loci (eQTL)
    ]{eqtl}{eQTL}{expression quantitative trait locus}
\newacronym{gtex}{GTEx}{the Genotype-Tissue Expression project}
\newacronym{gsmr}{GSMR}{generalized summary-based Mendelian Randomization}
\newacronym{ivw}{IVW}{inverse-variance weighted}
\newacronym{gefos}{GEFOS}{the Genetic Factors for Osteoporosis Consortium}

\newdualentry{mr}{MR}{Mendelian Randomization}{
    An approach based on randomized controlled trials used to infer the causal effect of an exposure to an outcome using genetic variants as \glspl{iv}.
    It requires the following assumptions to be met: I) the \glspl{iv} are associated with the exposure, II) the \glspl{iv} affect the outcome exclusively through the exposure, III) the \glspl{iv} are not associated with any confounders.
}

% START OF DOCUMENT
\begin{document}

\frontmatter
\newlength{\oldparindent}
\setlength{\oldparindent}{\parindent}

\parskip 6pt
\parindent 0pt

\begin{titlepage}
    \centering
    \makeatletter
    \LARGE \sffamily \@title

    \Large \rmfamily \@author

    \vspace*{\fill}
    \includegraphics[width=.5\pagewidth]{img/UU_logo_4f_42.pdf}

    \vspace*{\fill}
    \normalsize
    \textbf{Half-Time Thesis} \\
    Department of Immunology, Genetics and Pathology,\\
    Science for Life Laboratory, Uppsala University

    \@date
    \makeatother
\end{titlepage}

\textbf{Main Supervisor}\\
Ã…sa Johansson, PhD\\
Associate Professor \\
Department of Immunology, Genetics and Pathology. Medical Genetics and Genomics, Science for Life Laboratory, Uppsala University, Sweden

\textbf{Cosupervisors}\\
Torgny Karlsson, PhD\\
Researcher \\
Department of Immunology, Genetics and Pathology. Medical Genetics and Genomics, Science for Life Laboratory, Uppsala University, Sweden

Adam Ameur, PhD \\
Bioinformatician \\
Department of Immunology, Genetics and Pathology, Uppsala Genome Center, Uppsala University, Sweden

\textbf{Review Committee}
TBD

\printglossary[type=\acronymtype]

\printglossary

\chapter{Abstract}
    This is an abstract! And this is a \href{https://schmytzi.github.io/}{link}! There is also a \gls{mr} study.

\chapter{List of Publications}
This thesis is based on the following papers and projects, referred to in the text by their numbers.

\begin{enumerate}[label=\Roman*.]
    \item \fullcite{Schmitz2021}
    \item \fullcite{Johansson2021}
    \item Characterizing Copy Number Variations using Next- and Third-Generation Sequencing and their Association with Plasma Biomarkers
    \item Unspecified PacBio Project
\end{enumerate}

\section*{Related Publications}
The following publications are not part of the main research project.
\begin{itemize}
    \item \fullcite{Kierczak2021}
\end{itemize}

\tableofcontents

\mainmatter
\glsresetall
\parskip 0pt
\parindent \oldparindent
\chapter{Introduction}

\chapter{Aims}

\chapter{Project I}

\section{Rationale}
Estrogen, which is generally known as the primary female sex hormone, is responsible for the female reproductive system's development.
Furthermore, it regulates the menstrual cycle and plays a critical role in male sexual function \cite{Bates2013b,Hess1997b}. 
Among the three major forms of estrogen: estrone, estradiol and estriol, estradiol is the most potent and abundant \cite{Thomas2013c}.

Estradiol levels have been associated with several conditions, incl. deep vein thrombosis, cancers and \gls{t2d} \cite{Cauley1999a, Rosendaal2003b,Vikan2010}.
In particular, declining estradiol levels after menopause have been linked to reduced \gls{bmd} and, in turn, higher risk of osteoporosis \cite{Riggs1998a,Longo2012a}.

Previous \glspl{gwas} for estradiol levels have been performed in sex-stratified populations comprising up to 11,000 people, most often of European descent \cite{Pott2019e,Chen2013d,Liu2013b,Prescott2012f,Eriksson2018b}.
Additionally, a recent study in \gls{ukb} identified strong sex-specific genetic effects on testosterone but excluded associations with estradiol measurements because of their strong link to age at menopause \cite{Ruth2020d}.

Apart from \glspl{gwas}, the causal effect of hormones on diseases and disease risk has been assessed using \gls{mr} \cite{Eriksson2018b,Ruth2020d, Nethander2018a}.
While previous \gls{mr} studies identified a beneficial effect of estradiol on \gls{bmd} in males, there have been no such studies in females due to failure to identify valid instruments for \gls{mr} analyses \cite{Eriksson2018b, Nethander2018a}.

\section{Methods}
\Gls{ukb} is a long-time cohort study comprising about 500,000 participants born between 1939 and 1970 that were recruited from 2006 to 2010.
Extensive information on lifestyle and anthropometric traits was collected.
Participants left blood samples that were used for diverse measurements, including hormone levels, as well as genotyping of approximately 800,000 \glspl{snp}.

We used \gls{snp} data imputed using UK10K and 1000 genomes phase 3 reference panels, containing 93,093,070 \glspl{snp} overall.
We removed \glspl{snp} which had \gls{maf} $ < 0.01$, imputation quality $ < 0.3 $, more than 5\% missing phenotype data or deviated from Hardy-Weinberg equilibrium ($p < 10^{-20}$).
After \gls{qc}, 7,651,231 autosomal \glspl{snp} and 220,468 \glspl{snp} on chromosome~X remained. Furthermore, we restricted the analysis to unrelated Caucasian participants, excluding those with sex discordance, high heterozygosity/missingness and more than 5\% missing phenotypes. After \gls{qc}, 361,975 individuals remained, of which 167,168 were male and 194,807 were female.

Only measurements that were taken from blood samples given at the first visit at the assessment center were included in the analysis.
Estradiol was measured by two-step competitive analysis using a Beckman Coulter Unicel Dxl 800.
The assay had a lower detection limit of \qty{175}{\pmol\per\litre}, which is above the normal range for serum estradiol concentrations in postmenopausal females (0 -- \qty{73.4}{\pmol\per\litre}) \cite{Nakamoto2010a}.
Because of the resulting large fraction of measurements below detection limit, estradiol levels were analyzed as a binary phenotype (above/below detection limit).

We performed two sex-stratified \glspl{gwas} using logistic regression with additive genetic modeling in \textsf{PLINK~2}.
We included age, \gls{bmi}, the first ten genetic \glspl{pc} and the used genotyping array as covariates.
For females, we included \gls{hrt}, \gls{oc} use, number of live births, menopausal status and whether they had had a hysterectomy, too.
We identified lead \glspl{snp} by applying conditional analyses until no significant hits remained.
We tested all lead \glspl{snp} for sex-specific effects by including an interaction term in the logistic model.
We performed four sensitivity analyses.
We stratified females into pre- and postmenopausal, excluded all patients with cancer diagnoses and included testosterone and \gls{shbg} levels as covariates.
Lastly, we applied a Tobit-I model, which allowed us to incorporate quantitative estradiol measurements where available.

We annotated our lead \glspl{snp} to their closest genes and identified possible functional effects using \textsf{HaploReg} version 4.1 \cite{Ward2012}.
We used data from \gls{gtex} to check for overlap with known \glspl{eqtl} \cite{Carithers2015}.
Lastly, data from the GWAS Catalog was used to search for prior functional annotation of our lead \glspl{snp}.

To estimate the effect of estradiol on \gls{bmd}, we performed a one-sample \gls{mr} analysis in males and females separately.
\Gls{bmd} had been recorded using an ultrasound measurement and converted to T-Scores, i.e. the number of standard deviations the measurement differed from the patient's sex's mean. 

The main \gls{mr} analysis was performed using the \textsf{gsmr} package, version 1.0.8 \cite{Zhu2018}, which implements the \gls{gsmr} method.
We performed sensitivity analyses using the \gls{ivw}, weighted median and MR-Egger methods implemented in the \textsf{TwoSampleMR} package in \texttt{R} \cite{Hemani2018}.
Additionally, we performed a two-sample \gls{mr} test using the aforementioned methods with summary statistics for lumbar-spine \gls{bmd} from \gls{gefos} \cite{Estrada2012}.
Because \gls{gefos} had used a different imputation panel than \gls{ukb}, we revised the set of \glspl{iv}.
For each locus, we selected the most significant common \gls{snp} in \gls{ld} with our lead \gls{snp}.
\chapter{Project II}

\chapter{Project III}

\chapter{Work in Progress}

\chapter{Future Work}

\chapter{Concluding Remarks}


\backmatter

\chapter{Acknowledgements}
\printbibliography
\end{document}