\documentclass[draft]{scrbook}

% PACKAGES
\usepackage{typearea}
\usepackage{hyperref}
\usepackage[svgnames]{xcolor}
\usepackage{fontspec}
\usepackage{graphicx}
\usepackage{csquotes}
\usepackage[backend=biber, style=nature,maxcitenames=99, maxbibnames=99]{biblatex}
\usepackage[acronyms, toc]{glossaries}
\usepackage{enumitem}
\usepackage{xparse}
\usepackage{cleveref}
\usepackage{siunitx}
\usepackage{microtype}

% CUSTOM COMMANDS
\DeclareDocumentCommand{\newdualentry}{ O{} O{} m m m m } {
  \newglossaryentry{gls-#3}{name={#5},text={#5\glsadd{#3}},
    description={#6},#1
  }
  \makeglossaries
  \newacronym[see={[Glossary:]{gls-#3}},#2]{#3}{#4}{#5\glsadd{gls-#3}}
}

% SETTINGS
\definecolor{UppsalaRed}{RGB}{182,57,77}
\hypersetup{
    colorlinks=true,
    linkcolor=UppsalaRed,
    urlcolor=UppsalaRed,
    filecolor=UppsalaRed,
    citecolor=UppsalaRed
}
\defaultfontfeatures{Ligatures={Required, Common, TeX}}
\addbibresource{Half-Time Thesis.bib}
\DeclareSIUnit{\litre}{\ell}
\sisetup{
    per-mode = symbol
}

% METADATA
\title{The Effect of Common and Rare Genetic Variants on the Pathogenesis of Common Complex Diseases}
\author{Daniel Schmitz}
\date{2021-10-15}

% GLOSSARY
\makeglossaries
\newacronym{snp}{SNP}{single-nucleotide polymorphism}
\newacronym{cnv}{CNV}{copy-number variation}
\newacronym[
    plural=GWAS,
    firstplural=genome-wide association studies (GWAS)
    ]{gwas}{GWAS}{genome-wide association study}
\newacronym{ld}{LD}{linkage disequilibrium}
\newacronym{nsphs}{NSPHS}{Northern Swedish Population Health Study}
\newacronym{pea}{PEA}{protein extension assay}
\newacronym{ukb}{UKB}{UK Biobank}
\newacronym{wgs}{WGS}{whole-genome sequencing}
\newacronym{smrt}{SMRT}{Single-Molecule Real-Time Sequencing}
\newacronym{t2d}{T2D}{type 2 diabetes}
\newacronym{bmd}{BMD}{bone mineral density}
\newacronym{iv}{IV}{instrumental variable}
\newacronym{maf}{MAF}{minor allele frequency}
\newacronym{qc}{QC}{quality control}
\newacronym{bmi}{BMI}{body-mass index}
\newacronym{hrt}{HRT}{hormone-replacement therapy}
\newacronym{oc}{OC}{oral contraceptive}
\newacronym{pc}{PC}{principal component}
\newacronym{shbg}{SHBG}{sex hormone-binding globulin}
\newacronym[
    plural = eQTL,
    firstplural = expression quantitative trait loci (eQTL)
    ]{eqtl}{eQTL}{expression quantitative trait locus}
\newacronym{gtex}{GTEx}{the Genotype-Tissue Expression project}
\newacronym{gsmr}{GSMR}{generalized summary-based Mendelian Randomization}
\newacronym{ivw}{IVW}{inverse-variance weighted}
\newacronym{gefos}{GEFOS}{the Genetic Factors for Osteoporosis Consortium}
\newacronym{or}{OR}{odds ratio}
\newacronym{dhea}{DHEA}{dehydroepiandrosterone}
\newacronym{cyp3a7}{CYP3A7}{cytochrome P450 3A7}
\newacronym[
    first=the International Classification of Diseases
    ]{icd}{ICD}{International Classification of Diseases}
\newacronym{icd9}{ICD-9}{\gls{icd}, revision 9}
\newacronym[first=revision 10 (ICD-10)]{icd10}{ICD-10}{\gls{icd}, revision 10}

\newdualentry{mr}{MR}{Mendelian Randomization}{
    An approach based on randomized controlled trials used to infer the causal effect of an exposure to an outcome using genetic variants as \glspl{iv}.
    It requires the following assumptions to be met: I) the \glspl{iv} are associated with the exposure, II) the \glspl{iv} affect the outcome exclusively through the exposure, III) the \glspl{iv} are not associated with any confounders.
}

% START OF DOCUMENT
\begin{document}

\frontmatter
\newlength{\oldparindent}
\setlength{\oldparindent}{\parindent}

\parskip 6pt
\parindent 0pt

\begin{titlepage}
    \centering
    \makeatletter
    \LARGE \sffamily \@title

    \Large \rmfamily \@author

    \vspace*{\fill}
    \includegraphics[width=.5\pagewidth]{img/UU_logo_4f_42.pdf}

    \vspace*{\fill}
    \normalsize
    \textbf{Half-Time Thesis} \\
    Department of Immunology, Genetics and Pathology,\\
    Science for Life Laboratory, Uppsala University

    \@date
    \makeatother
\end{titlepage}

{ % Nested in a group to reset typesetting
    \raggedright
    \textbf{Main Supervisor}\\
    Åsa Johansson, PhD\\
    Associate Professor \\
    Department of Immunology, Genetics and Pathology. Medical Genetics and Genomics, Science for Life Laboratory, Uppsala University, Sweden

    \textbf{Co-Supervisors}\\
    Torgny Karlsson, PhD\\
    Researcher \\
    Department of Immunology, Genetics and Pathology. Medical Genetics and Genomics, Science for Life Laboratory, Uppsala University, Sweden

    Adam Ameur, PhD \\
    Bioinformatician \\
    Department of Immunology, Genetics and Pathology, Uppsala Genome Center, Uppsala University, Sweden

    \textbf{Review Committee}\\
    TBD

    % COPYRIGHT NOTICE
    \vfill
    \footnotesize
    © 2021 Daniel Schmitz

    This work is licensed under the Creative Commons Attribution 4.0 International License. To view a copy of this license, visit \url{http://creativecommons.org/licenses/by/4.0/}.
}

\chapter{Abstract}
    This is an abstract! And this is a \href{https://schmytzi.github.io/}{link}! There is also a \gls{mr} study.

\chapter{List of Publications}
This thesis is based on the following papers and projects, referred to in the text by their numbers.

\begin{enumerate}[label=\Roman*.]
    \item \fullcite{Schmitz2021}
    \item \fullcite{Johansson2021}
    \item Characterizing Copy Number Variations using Next- and Third-Generation Sequencing and their Association with Plasma Biomarkers
    \item Unspecified PacBio Project
\end{enumerate}

\section*{Related Publications}
The following publications are not part of the main research project.
\begin{itemize}
    \item \fullcite{Kierczak2021}
\end{itemize}

\tableofcontents

\mainmatter
\glsresetall
\parskip 0pt
\parindent \oldparindent
\chapter{Introduction}

\chapter{Aims}

\chapter{Project I}

\section{Background}
Estrogen, which is generally known as the primary female sex hormone, is responsible for the female reproductive system's development.
Furthermore, it regulates the menstrual cycle and plays a critical role in male sexual function \cite{Bates2013b,Hess1997b}. 
Among the three major forms of estrogen: estrone, estradiol and estriol, estradiol is the most potent and abundant \cite{Thomas2013c}.

Estradiol levels have been associated with several conditions, incl. deep vein thrombosis, cancers and \gls{t2d} \cite{Cauley1999a, Rosendaal2003b,Vikan2010}.
In particular, declining estradiol levels after menopause have been linked to reduced \gls{bmd} and, in turn, higher risk of osteoporosis \cite{Riggs1998a,Longo2012a}.

Previous \glspl{gwas} for estradiol levels have been performed in sex-stratified populations comprising up to 11,000 people, most often of European descent \cite{Pott2019e,Chen2013d,Liu2013b,Prescott2012f,Eriksson2018b}.
Additionally, a recent study in \gls{ukb} identified strong sex-specific genetic effects on testosterone but excluded associations with estradiol measurements because of their strong link to age at menopause \cite{Ruth2020d}.

Apart from \glspl{gwas}, the causal effect of hormones on diseases and disease risk has been assessed using \gls{mr} \cite{Eriksson2018b,Ruth2020d, Nethander2018a}.
While previous \gls{mr} studies identified a beneficial effect of estradiol on \gls{bmd} in males, there have been no such studies in females due to failure to identify valid instruments for \gls{mr} analyses \cite{Eriksson2018b, Nethander2018a}.

\section{Methods} \label{p1methods}
\Gls{ukb} is a long-time cohort study comprising about 500,000 participants born between 1939 and 1970 that were recruited from 2006 to 2010.
Extensive information on lifestyle and anthropometric traits was collected.
Participants left blood samples that were used for diverse measurements, including hormone levels, as well as genotyping of approximately 800,000 \glspl{snp}.

We used \gls{snp} data imputed using UK10K and 1000 genomes phase 3 reference panels, containing 93,093,070 \glspl{snp} overall.
We removed \glspl{snp} which had \gls{maf} $ < 0.01$, imputation quality $ < 0.3 $, more than 5\% missing phenotype data or deviated from Hardy-Weinberg equilibrium ($p < 10^{-20}$).
After \gls{qc}, 7,651,231 autosomal \glspl{snp} and 220,468 \glspl{snp} on chromosome~X remained. Furthermore, we restricted the analysis to unrelated Caucasian participants, excluding those with sex discordance, high heterozygosity/missingness and more than 5\% missing phenotypes. After \gls{qc}, 361,975 individuals remained, of which 167,168 were male and 194,807 were female.

Only measurements that were taken from blood samples given at the first visit at the assessment center were included in the analysis.
Estradiol was measured by two-step competitive analysis using a Beckman Coulter Unicel Dxl 800.
The assay had a lower detection limit of \qty{175}{\pmol\per\litre}, which is above the normal range for serum estradiol concentrations in postmenopausal females (0 -- \qty{73.4}{\pmol\per\litre}) \cite{Nakamoto2010a}.
Because of the resulting large fraction of measurements below detection limit, estradiol levels were analyzed as a binary phenotype (above/below detection limit).

We performed two sex-stratified \glspl{gwas} using logistic regression with additive genetic modeling in \textsf{PLINK~2}.
We included age, \gls{bmi}, the first ten genetic \glspl{pc} and the used genotyping array as covariates, as well as a binary indicator for the used genotyping array to control for batch effects.
For females, we included \gls{hrt}, \gls{oc} use (never/ever/current), number of live births, menopausal status and whether they had had a hysterectomy, too.
We identified lead \glspl{snp} by applying conditional analyses until no significant hits remained.
We tested all lead \glspl{snp} for sex-specific effects by including an interaction term in the logistic model.
We performed four sensitivity analyses.
We stratified females into pre- and postmenopausal, excluded all patients with cancer diagnoses and included testosterone and \gls{shbg} levels as covariates.
Lastly, we applied a Tobit-I model, which allowed us to incorporate quantitative estradiol measurements where available.

We annotated our lead \glspl{snp} to their closest genes and identified possible functional effects using \textsf{HaploReg} version 4.1 \cite{Ward2012}.
We used data from \gls{gtex} to check for overlap with known \glspl{eqtl} \cite{Carithers2015}.
Lastly, data from the GWAS Catalog was used to search for prior functional annotation of our lead \glspl{snp}.

To estimate the effect of estradiol on \gls{bmd}, we performed a one-sample \gls{mr} analysis in males and females separately using our \gls{gwas} lead \glspl{snp} as \glspl{iv}.
Due to the low number of significant associations in the female cohort, we applied a relaxed significance threshold ($p < 10^{-7}$) tp increase the number of available \glspl{iv}.
\Gls{bmd} had been recorded using an ultrasound measurement and converted to T-Scores, i.e. the number of standard deviations the measurement differed from the patient's sex's mean. 

The main \gls{mr} analysis was performed using the \textsf{gsmr} package, version 1.0.8 \cite{Zhu2018}, which implements the \gls{gsmr} method.
We performed sensitivity analyses using the \gls{ivw}, weighted median and MR-Egger methods implemented in the \textsf{TwoSampleMR} package in \texttt{R} \cite{Hemani2018}.
Additionally, we performed a two-sample \gls{mr} test using the aforementioned methods with summary statistics for lumbar-spine \gls{bmd} from \gls{gefos} \cite{Estrada2012}.
Because \gls{gefos} had used a different imputation panel than \gls{ukb}, we revised the set of \glspl{iv}.
For each locus, we selected the most significant common \gls{snp} in \gls{ld} with our lead \gls{snp}.

\section{Results}

After genotype and estradiol \gls{qc}, 147,690 males remained, of whom 134,323 had estradiol below and 13,367 above detection limit (\qty{175}{\pmol\per\l}).
Slightly more females (163,985) remained, with 126,524 individuals having estradiol levels below detection limit and 37,461 above.
Only 9.1\% of males and 7.9\% of postmenopausal females had detectable estradiol levels, while 71.9\% of premenopausal females had measurements.

We found 15 loci on 14 chromosomes to be significantly associated ($p < 5*10^{-8}$) with estradiol levels, of which 13 were specific to males, one (\textit{MCM8}) specific to females and one (\textit{CYP3A7}) shared between both sexes.
We identified one conditional hit each on chromosomes 2 and 15.
12 of our \gls{gwas} hits had already established links to estradiol or steroid-hormone metabolism.

After stratification of the female cohort into pre- and postmenopausal, we identified one locus (\textit{CYP3A7}) with significantly different effects between the two strata.
Removal of all participants with previous cancer diagnoses did not lead to a change in our primary \gls{gwas} results.
Lastly, adjusting our model for testosterone and \gls{shbg} levels caused the loci \textit{SHBG} and \textit{FKBP4} to lose genome-wide significance with lower effect estimates.
The loci \textit{AR} and \textit{UGT3A1} lost significance, too, but the estimated \glspl{or} did not differ significantly.
In the quantitative \gls{gwas} using Tobit modeling, 3 \glspl{snp} lost genome-wide significance.

\section{Discussion}
We performed a \gls{gwas} for dichotomized estradiol levels for males and females in \gls{ukb}.
There were 15 genome-wide significant loci, 14 of which were male-specific, one female-specific and one shared between sexes.
An additional analysis using a sex-genotype interaction term revealed strong sex-specific effects.
We found two loci on chromosomes 2 and 15 to be independently associated with estradiol measurements independent of the lead \glspl{snp} on their respective chromosome.
Most of the loci we identified have previously established links to steroid-hormone metabolism, including synthesis, conversion, transport and elimination of steroid hormones.

\textit{CYP3A7} was significant in both males and females, indicating an important role in estrogen metabolism in both sexes.
Interestingly, it was also the only gene to have significantly different effects in pre- and postmenopausal females.
\textit{CYP3A7} encodes \gls{cyp3a7}, which metabolizes a precursor of both androgens and estrogens: \gls{dhea} \cite{Ohmori1998}.
Up to 75\% of estrogens in premenopausal women are derived from \gls{dhea} and after menopause, it is the main precursor of androgens and estrogens \cite{Simpson2001}.
When adjusting for \gls{shbg} and testosterone, \textit{CYP3A7}'s effect disappeared in females.
These findings point to \gls{cyp3a7} fulfilling different functions for steroid-hormone metabolism in males and females.

Interestingly, \textit{ABO}, the gene responsible for the ABO blood groups, was associated with estradiol levels in males \cite{Ogasawara1996}.
Our effect allele (rs657152-A) is in \gls{ld} with rs8176719-G, which is present in individuals that do not have blood type O.
This indicates that people with blood type O have higher estradiol levels.

We estimated the causal effect of estradiol on \gls{bmd} using \gls{mr} in males and for the first time in females.
We included up to 16 \glspl{snp} in our \gls{mr} analyses, a large increase from five \glspl{iv} from previous studies \cite{Nethander2018a}.
Our effect estimates were higher in females than in males, indicating that bone metabolism depends more on estradiol in females.
This ag<rees with the rapid decline of \gls{bmd} after menopause and subsequently the prevalence of osteoporosis in postmenopausal women.

While transforming estradiol measurements into a binary phenotype enabled us to increase our cohort size, doing so prohibited us from capturing the full variation in the cohort.
Therefore, we used applied a Tobit-I model, which incorporates quantitative measurements where available without discarding individuals below detection limit.
The effect sizes estimated by this method were comparable with those from quantitative studies in the GWAS Catalog.
This method had not been used for \glspl{gwas} before, but our results indicate that it should be useful for future studies.

We identified few significant loci in females, which led to only four \glspl{iv} being included in the \gls{mr}.
Estradiol levels in women vary wildly during the menstrual cycle, making the genetic effect hard to estimate.
Furthermore, estrogen levels drop after menopause and are mostly determined by the time that has passed since the last menstruation \cite{Richardson2020}.
Both of these aspects probably limited the power of our \gls{gwas}.
Moreover, the low number of \glspl{iv} in our \gls{mr} analyses could have made the results unstable and weak to pleiotropy.

In summary, we identified genetic loci that affect estradiol levels with strong sex-dependent effects.
We showed the causal effect of estradiol on \gls{bmd}, supporting \gls{hrt} as a preventative treatment of osteoporosis.
Our findings confirm established medical research as well as provide insight into the metabolism and function of estrogens.

\chapter{Project II}
\section{Background}
Despite its important functions for development and health, estrogen has been associated with a number of diseases.
Higher estradiol levels have been associated with an increased risk of breast cancer in pre- as well as postmenopausal women \cite{Key2013,Kaaks2005,Zhang2013,Kaaks2005a}.
However, a definite causal relationship, i.e. whether high estradiol levels increase the risk of breast cancer or cancer progression causes estradiol levels to rise, has not been established.

Two other major forms of cancer---endometrial and ovarian cancer---have clearly established links to estradiol levels \cite{Brinton2014,Mungenast2014}.
Progesterone has been shown to have a protective effect against these kinds of cancer, which is why menopausal \gls{hrt} is often combined with progesterone.
The same effect can be observed for \glspl{oc} because they contain a synthetic form of progesterone---progestin \cite{Karlsson2021,Iversen2017}.

Despite the clearly established association between the aforementioned cancers and estrogen, it remains unclear whether the body's own production of estrogen has an effect on cancer risk.
\gls{mr} is a commonly used approach to establish such a causal link.
In fact, there has been one study that used one \gls{snp} as \gls{iv} to estimate the effect of endogenous estradiol on endometrial cancer \cite{Thompson2016}.
No previous \gls{mr} studies concerning the effect of estradiol on breast or ovarian cancer have been published.

\section{Methods}
We used female participants from \gls{ukb} as the base cohort for this study.
Genotype and sample \gls{qc} were discussed in \hyperref[p1methods]{Project~I}, as we used the effect estimates for estradiol from that study.

We assessed cancer incidence using diagnoses from hospital stays, death and cancer registries as well as answers from verbal interviews and touchscreen questionnaires.
Diagnoses were encoded as codes from \gls{icd9} and \gls{icd10}.
Breast cancer was represented by codes 174 (\gls{icd9}) and C50 (\gls{icd10}), ovarian cancer by codes 183 (\gls{icd9}) and C56 (\gls{icd10}) and endometrial cancer by \gls{icd10} code C541.
Most cases were recorded in the cancer register.
However, data from before 1995 was lacking.
Therefore, we had to rely on self-reported data for these cases.

We estimated the effects of our estradiol \glspl{iv} on risk for all three cancers using \textsf{PLINK 2}.
We applied a logistic model with additive effects with the following covariates: age, \gls{bmi}, genetic \glspl{pc} 1--10, \gls{hrt} use (never/ever/current), \gls{oc} use (never/ever/current), number of live births, menopausal status, whether the participant had undergone a hysterectomy and a binary indicator for the used genotyping array.

To infer the causal effect of endogenous estradiol on cancer risk, we performed a one-sample \gls{mr} analysis using our computed effect estimates and a two-sample \gls{mr} analysis, where we used publicly available \gls{gwas} data for \gls{snp} effects on cancer risk.
The main analyses were performed using the \texttt{R} package \textsf{gsmr} version 1.0.8 \cite{Zhu2018}.
We used the \texttt{R} package \textsf{MendelianRandomization} for sensitivity analyses, using the included methods: robust \gls{ivw}, weighted median and MR-Egger \cite{Olena2017}.


\chapter{Project III}

\chapter{Concluding Remarks}


\backmatter

\chapter{Acknowledgements}

\printglossary[type=\acronymtype]

\printglossary

\printbibliography
\end{document}